src/ethereum_test_tools/common/types.py

```diff
@@ -425,6 +448,24 @@ def with_code(cls: Type, code: bytes | str | Code) -> "Account":
         return Account(nonce=1, code=code)
 
 
+def alloc_to_accounts(got_alloc: Dict[str, Any]) -> Mapping[str, Account]:
+    """
+    Converts the post state alloc returned from t8n to a mapping of accounts.
+    """
+    accounts = {}
+    for address, value in got_alloc.items():
+        account = Account(
+            nonce=int(value.get("nonce", None), 16)
+            if "nonce" in value
+            else None,
```

Comment:
I think we could use `int_or_none` here, but it depends on whether the value returned here from t8n has the `0x` at the beginning or not.

Reply:
Sounds good!